### PR TITLE
silence accounts from global community page

### PR DESCRIPTION
**Pull request by tobiasd**
The community page can show all public postings arriving on a node. Some users might follow someone who posts content one does not want to see on the community page of the node. This script acts as a CLI interface to silence accounts from the global community page of the node.

Currently this has to be done directly at the database.

### CODE REVIEW

File: util/global_community_silence.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env php
+<?php
+
+/**
+ * @brief tool to silence accounts on the global community page
+ *
+ * With this tool, you can silence an account on the global community page.
+ * Postings from silenced accounts will not be displayes on the community
```
Comment (MrPetovan) [inline util/global_community_silence.php]: “displayed”

File: util/global_community_silence.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ * page. This silencing does only affect the display on the community page,
+ * accounts following the silenced accounts will still get their postings.
+ *
+ * Usage: pass the URL of the to be silenced account as only parameter
+ *        at the command line when running this tool. E.g.
+ *
+ *        $> util/global_community_silence.php http://example.com/profile/bob
+ *
+ *        will silence bob@example.com so that his postings wont appear at
```
Comment (MrPetovan) [inline util/global_community_silence.php]: “won’t”

File: util/global_community_silence.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+/**
+ * @brief tool to silence accounts on the global community page
+ *
+ * With this tool, you can silence an account on the global community page.
+ * Postings from silenced accounts will not be displayes on the community
+ * page. This silencing does only affect the display on the community page,
+ * accounts following the silenced accounts will still get their postings.
+ *
+ * Usage: pass the URL of the to be silenced account as only parameter
```
Comment (MrPetovan) [inline util/global_community_silence.php]: + profile/user

File: util/global_community_silence.php [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ *        will silence bob@example.com so that his postings wont appear at
+ *        the global community page.
+ *
+ * Author: Tobias Diekershoff
+ *
+ * License: AGPLv3 or later, same as Friendica
+ **/
+
+if ($argc!=2 || $argv[1]=="-h" || $argv[1]=="--help" || $argv[1]=="-?") {
```
Comment (MrPetovan) [inline util/global_community_silence.php]: Standards: Please add spaces around operators.

File: util/global_community_silence.php [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+	echo "Example: Silence bob@example.com\r\n";
+	echo "$> ".$argv[0]." https://example.com/profiles/bob\r\n";
+	echo "\r\n";
+	exit(0);
+}
+
+use Friendica\Database\DBM;
+use Friendica\Network\Probe;
+
+require_once("boot.php");
```
Comment (MrPetovan) [inline util/global_community_silence.php]: Standards: Please remove parentheses from language constructs.

File: util/global_community_silence.php [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+require_once('include/dba.php');
+require_once("include/text.php");
```
Comment (MrPetovan) [inline util/global_community_silence.php]: Standards: Please decide between single or double-quotes for the whole file. Single quotes are preferred.

**Comment by annando** [comment]
I guess that @tobiasd should have a look at it. I totally overlooked that.

**Pull request merged**